src/screens/Customers/CustomerMain.tsx

```diff
@@ -83,23 +130,43 @@ function CustomerMain(props: RouteComponentProps & UserProps) {
     setFilteredCustomers(customers);
   }
 
+  const handleMenuSelect = (e: any) => {
+    const keys: string[] = e.target.value; 
+    const key: string = keys[keys.length - 1]; 
```

Comment:
I think this naming convention might be a bit confusing for people reading the code after the fact. Maybe we could be more specific with these variable names?

like `selectedOptions` and `newlySelectedOption` or something along those lines? Otherwise I think it becomes unclear why the last key is taken from the list (it also might be good to comment why the last key is chosen for those who look at this code later and wonder)

Reply:
Gotcha, I'll write a comment for this.